src/page/Home.jsx

```diff
@@ -205,6 +209,7 @@ const Home = () => {
                   ARCHIVED
                 </ToggleButton>
               </HomeToggleButtonGroup>
+              {/* todo: filter */}
```

Comment:
sorry, what is this filter for!

Reply:
The filter function supposed to be for home page can see from design, but not high priority to develop now